editor/components/document-outline/index.js

```diff
@@ -63,6 +63,13 @@ export const DocumentOutline = ( { blocks = [], onSelect } ) => {
 	// Select the corresponding block in the main editor
 	// when clicking on a heading item from the list.
 	const onSelectHeading = ( uid ) => onSelect( uid );
+	const focusTitle = () => {
+		// Not great but it's the simplest way to focus the title right now.
```

Comment:
Any thoughts on how we might be able to improve this?

Maybe focus becoming managed reducer state, and the title component listening via `connect` for changing props?

Reply:
I think the current code is better than doing this. It seems overkill to me. The approach you propose is more in line with the unidirectional data flow but I prefer to be pragmatic here.

Reply:
One way to improve is to export the className as a constant of the PostTitle component (same way we do for the `BlockListBlock` component.

Reply:
I wouldn't optimize too much for title being outside of the block-list, as we want to bring it in at some point.